Sources/App/Views/PublicPage.swift

```diff
@@ -201,7 +203,7 @@ class PublicPage {
                 .a(
                     .href(SiteURL.home.relativeURL()),
                     .h1(
-                        .img(.src(SiteURL.images("logo.svg").relativeURL())),
+                        .img(.src(SiteURL.images("logo.svg").absoluteURL())),
```

Comment:
Actually, now that I see this: would could use the same mechanism we use for stylesheets to avoid this change here as well.

Reply:
Done